src/client/components/content-section/content-section.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <section :id="section.slug">
+  <section class="page-content" :id="section.slug">
```

Comment:
We always name the root element after the component, so `class="content-section"`